Jenkinsfile

```diff
@@ -30,8 +30,13 @@ properties([
                  description: 'Appends "-${GRAVITY_VERSION}" to the tag to be published'),
     booleanParam(name: 'IMPORT_APP',
                  defaultValue: false,
-                 description: 'Import application to ops center'
-    ),
+                 description: 'Import application to ops center'),
+    string(name: 'S3_UPLOAD_PATH',
+           defaultValue: '',
```

Comment:
No default value?

I'm not sure exactly how MS prefers to manage stuff like this, but you may consider making it a secret instead of a parameter that must be filled in manually each time.

Reply:
The upload could happen only if you pass `IMPORT_APP_PACKAGE` env variable.